src/components/IconButton/README.md

```diff
@@ -0,0 +1,4 @@
+## IconButton
```

Comment:
We should probably have a more informative README.
You can look at the other buttons for instance.